src/main/java/org/gridsuite/actions/server/FiltersToGroovyScript.java

```diff
@@ -80,15 +80,15 @@ public String generateGroovyScriptFromFilters(FiltersContingencyListAttributes f
         ST template = new ST(script);
 
         template.add("collectionName", equipmentsCollection);
+        template.add("nominalVDefined", filtersContingencyListAttributes.getNominalVoltage() != -1);
```

Comment:
Was it really necessary to introduce nominalVDefined ? why not just not defined nominalV

Reply:
Done